src/OpenSheetMusicDisplay/OSMDOptions.ts

```diff
@@ -293,3 +297,27 @@ export interface AutoBeamOptions {
      */
     groups?: [number[]];
 }
+
+export interface CursorOptions {
+    /**
+     * Type of cursor:
+     * 0: Standard highlighting current notes
+     * 1: Thin line left to the current notes
+     * 2: Short thin line on top of stave and left to the current notes
+     * 3: Current compass
+     * 4: Current compass to left of current notes
+     */
+    type: number;
```

Comment:
I would prefer an enum here with named members. You can still do it like this and use numbers:
```ts
export enum CursorType {
  Standard: 0,
  ThinLeft: 1,
  ShortThinTopLeft: 2,
  CurrentArea,
  CurrentAreaLeft
}
```

If you find better names for these, that's fine.
By the way, what does "compass" mean? That has absolutely no meaning for me in the context of music. Is it the measure? The currently playing notes? I'd really like a more obvious name here.

Reply:
Name are find. You are right I refered to measure, 'compas' is spanish :). Also changed.

Reply:
done